htlcswitch/test_utils.go

```diff
@@ -1133,6 +1133,7 @@ func (h *hopNetwork) createChannelLink(server, peer *mockServer,
 		return nil
 	}
 
+	//nolint:lll
```

Comment:
maybe move this to the offending line(s)?

Reply:
`gofmt` pushes about half the lines over the 80 limit when the new line is added, so cleaner to leave as-is imo. 

Could also go with a shorter function name if we want to bow down the 80 line lords?

Reply:
😔